cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -366,6 +366,26 @@ runGenesisCreate (GenesisDir rootdir)
                        (Lovelace 10) (Lovelace 1, Lovelace 1) (1,1) (1,1) 1 1 1
 
   writeShelleyGenesis (rootdir </> "genesis.json") finalGenesis
+
+  liftIO $ LBS.writeFile (rootdir </> "genesis.alonzo.json") $ Aeson.encode $
+    Aeson.object
+    [ "adaPerUTxOWord" .= Aeson.toJSON @Int 0
+    , "executionPrices" .= Aeson.object
+      [ "prMem" .= Aeson.toJSON @Int 1
+      , "prSteps" .= Aeson.toJSON @Int 1
+      ]
+    , "maxTxExUnits" .= Aeson.object
+      [ "exUnitsMem" .= Aeson.toJSON @Int 1
+      , "exUnitsSteps" .= Aeson.toJSON @Int 1
+      ]
+    , "maxBlockExUnits" .= Aeson.object
+      [ "exUnitsMem" .= Aeson.toJSON @Int 1
+      , "exUnitsSteps" .= Aeson.toJSON @Int 1
+      ]
+    , "maxValueSize" .= Aeson.toJSON @Int 1000
+    , "collateralPercentage" .= Aeson.toJSON @Int 100
+    , "maxCollateralInputs" .= Aeson.toJSON @Int 1
+    ]
```

Comment:
`cardano-cli genesis create` now creates `<rootdir>/genesis.alonzo.json`.

Reply:
should `create-staked` also do that?